docs/java/features/connectivity/destinations.md

```diff
@@ -29,17 +29,38 @@ DestinationAccessor.getDestination("my-destination");
 This will look up the destination in the destination service if the application is running on SAP Cloud Platform. But
 also, other sources like the environment variables are considered.
 
+## Decorating Destinations ##
+Depending on the use case, one needs to wrap the accessed destination before making a request to any system. This is to make sure all required destination properties are correctly set before invoking the actual request. 
+
+### HTTP Destinations
+In case of HTTP connections one needs to wrap the retrieved destination as `DefaultHttpDestination` using `asHttp()`: 
```

Comment:
```suggestion
In case of HTTP connections one needs to wrap the retrieved destination as `HttpDestination` using `asHttp()`:
```

Reply:
For correctness, it actually doesn't have to be the `DefaultHttpDestination`, but any implementation of the interface `HttpDestination` will do.

Reply:
ok